stored_requests/fetcher.go

```diff
@@ -190,8 +191,17 @@ func (f *fetcherWithCache) FetchRequests(ctx context.Context, requestIDs []strin
 	return
 }
 
-func (f *fetcherWithCache) FetchAccount(ctx context.Context, accountID string) (json.RawMessage, []error) {
-	return f.fetcher.FetchAccount(ctx, accountID)
+func (f *fetcherWithCache) FetchAccount(ctx context.Context, accountID string) (account json.RawMessage, errs []error) {
+	accountData := f.cache.Accounts.Get(ctx, []string{accountID})
+	// TODO: add metrics
```

Comment:
You mentioned the metrics will be a separate PR right? And you're also planning to add account HTTP and DB fetchers in their own PRs too?

Reply:
The metrics commit is ready, I can make a PR right after this, or add another commit here.
Account http is almost ready, will make a PR separately.
I will not have time this round for the Db fetcher, but I'll try to take a stab at it.